### DISCUSSION TITLE
I can‘t change my windows power strategy with legion toolkit

### DISCUSSION THREAD

**Discussion by ToniCui** [BartoszCichecki/LenovoLegionToolkit#1073] [Q&A] [answered]
First of all, thanks a lot for making legion toolkit, it's much better than the original legion zone.

Besides, I wanna apologize for using Chinese in my screenshot, becasue the system on this laptop is customized for China and I can only use Chinese in Windows, for the same reason, I translate some keywords by myself, they might not be accurate, so if you're cofused please ask me.

My laptop is called  legion r9000p, like the system, it's a special model for China, I'll say it's kinda like Legion 5 and I'll put a screenshot with my hardwire info beneath. Now matter I choose what power mode in legion toolkit, the power plan on Windows remain quiet model/ high-power efficiency mode, as you can see in my second screenshot. Is it just I didn't change the right option or what?
<img width="343" alt="微信截图_20231210114045" src="https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/76056320/4e297f8b-1f2a-48db-afac-41f459264ce9">
<img width="998" alt="微信截图_20231210113651" src="https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/76056320/9cc0f7ec-3d55-40cb-b8d4-36f40538cc77">

**Comment by BartoszCichecki** [comment, marked as answer]
The power mode in LLT is not the same as the power mode in Windows. They are two completely separate settings.

**Reply by BartoszCichecki** [reply]
But to be fair I have been considering to add an option to synchronize these for a while now, so maybe the time is now :)

**Reply by ToniCui** [reply]
Thanks for you reply, now I fully understand why. And if you can add the synchronize function, that would be so helpful and perfect, as I won't need to change the Windows power plan manually every time I wanna play a game xd.

Any way thanks for replying and make this magnificant software! QvQ
